recipes/libsndio/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "1.9.0":
+    "source":
+      url: "https://github.com/ratchov/sndio/archive/refs/tags/v1.9.0.tar.gz"
+      sha256: "4e8d77069b651d7c4bb2a50ba49004bd7fcfdfec1516f88a76457f831c17a546"
+    "license":
+      url: "https://raw.githubusercontent.com/ratchov/sndio/e10e4f836b88f8d5ea2864845e3712f560e6bed6/LICENSE"
+      sha256: "4ae392ab9cfbf2d6ac8522f5f4b025685d60c824979fad783aaf72d55165730f"
```

Comment:
You'll want to copy the license out of t he downloaded source code instead. That's the typical idiom on CCI.

Reply:
It doesn't exist in any distribution of the source by the authors. They added the license to the git repo 9 months ago but the latest release is from 2022 so it doesn't contain it. The tarball on the authors website also does not contain the license.

The license is partially present in the actual source code files as well, but the authors are spread out across several different files.

I found the approach I took here in a number of CCI recipes, such as in bazel here:

https://github.com/conan-io/conan-center-index/blob/master/recipes/bazel/all/conandata.yml

Reply:
Ah, I didn't realize it wasn't yet in a release. Thanks!